src/sagemaker/serve/model_server/torchserve/inference.py

```diff
@@ -110,5 +117,30 @@ def _pickle_file_integrity_check():
     perform_integrity_check(buffer=buffer, metadata_path=metadeata_path)
 
 
+def _get_mlflow_flavor():
+    mlflow_model_flavor = os.getenv("MLFLOW_MODEL_FLAVOR")
+    return mlflow_model_flavor
+
+
+def _load_mlflow_model(deployment_flavor, model_dir):
+    flavor_loader_map = {
```

Comment:
Will we need to update this map?

Reply:
Maybe, since `langchain` flavor is still experimental and we might want to extend to new flavors in the future